ansible_collections/arista/avd/roles/eos_designs/docs/tables/network-services-vrfs-l3-interfaces-settings.md

```diff
@@ -132,13 +132,25 @@
                   cost: <int>
                   authentication: <str; "simple" | "message-digest">
 
-                  # Password used with simple authentication.
+                  # Key for OSPF simple authentication.
+                  # Expects a cleartext password if `encrypt_passwords: true`, otherwise expects a type 7 password.
+                  # Maximum 8 characters for cleartext password.
+                  #
+                  # NOTE: when using type 7 password, the l3_interfaces.interfaces list must not be more than 1 interface
+                  #       or they must all be the same (e.g. [Ethernet7, Ethernet7]) as the type7 password depends on the interface
+                  #       name (when common encryption is disabled).
                   simple_auth_key: <str>
                   message_digest_keys:
                     - id: <int>
                       hash_algorithm: <str; "md5" | "sha1" | "sha256" | "sha384" | "sha512"; default="sha512">
 
-                      # Key password.
+                      # Key for OSPF message digest.
+                      # Expects a cleartext password if `encrypt_passwords: true`, otherwise expects a type 7 password.
+                      # Maximum 16 characters for cleartext password.
+                      #
+                      # NOTE: when using type 7 password, the l3_interfaces.interfaces list must not be more than 1 interface
+                      # or they must all be the same (e.g. [Ethernet7, Ethernet7]) as the type7 password depends on the interface name
+                      # (when common encryption is disabled).
```

Comment:
This last comment is confusing when we also have VRF `encrypt_passwords` option in mind. I know you are thinking about EOS common encryption but we don't support that in eos_designs, so I am not sure we need to mention that here.